src/Components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -21,27 +21,27 @@ interface ComponentDetails {
 interface BasicSelectProps {
   componentDetails: ComponentDetails;
   options: { [key: string]: string };
-  householdData: HouseholdData;
-  setHouseholdData: any;
+  memberData: HouseholdData;
+  setMemberData: any;
```

Comment:
I think that we can get rid of this `any`
```suggestion
  setMemberData: (member: HouseholdData) => void;
```